docs/1.12/03-Tutorials2/06-Deploy-Prisma-Servers/06-Zeit-Now-and-Google-Cloud-SQL.md

```diff
@@ -0,0 +1,158 @@
+---
+alias: joofei3ahd
+description: Learn how to deploy Prisma servers to Zeit Now.
+---
+
+# Zeit Now and GCP Cloud SQL
+
+In this tutorial, you’re going to learn how to deploy a Prisma server to [Zeit Now](https://zeit.co/now). The server will be backed by a MySQL database hosted in Google Cloud Platform (GCP). If you wish to host your database elsewhere such as AWS RDS just skip directly to [part 2]().
+
+## Overview
+
+[Zeit Now](https://zeit.co/now) provides real-time Node and Docker cloud deployments. 
+
+Google Cloud SQL provides fully-managed PostgreSQL & MySQL instances. You may follow the [tutorials and quick guides](https://cloud.google.com/sql/docs/) to configure and setup your SQL instance using either the console or command line.
+
+## Prerequisites
+
+#### GCP Cloud SQL
+
+For the purpose of this tutorial you will need to have signed up for GCP, created a project and enabled the Cloud SQL Admin API (see [quickstart](https://cloud.google.com/sql/docs/mysql/) for more information).
+
+You also should install the [Cloud SDK](https://cloud.google.com/sdk/).
+
+#### Zeit Now
+
+Sign up to [zeit.co](https://zeit.co).
+
+Install the [now-cli)](https://github.com/zeit/now-cli)
+
+## 1. MySQL
+
+### 1.1. Creating A Cloud SQL Instance
+
+We'll be creating a MySQL instance called `prisma` on GCP via the command line. You may also do this via the console.
+
+```console
+❯ gcloud sql instances create prisma --tier=db-f1-micro --authorized-networks=0.0.0.0/0 --region=europe-west1
```

Comment:
ideally the bash command and the output are separated into two code blocks. then you can remove the `❯` character in front of the bash command which allows people to directly copy the code from the code block and paste it into the terminal